scripts/npm/update-owners.sh

```diff
@@ -18,11 +18,13 @@
 #
 set -e
 
-cat NPM_OWNERS
+NPM_OWNERS="$( node scripts/npm/get-owners.js )"
+
+echo "$NPM_OWNERS"
 read -p "Do you want to add the above owners to all packages (y/n)? " confirm
 
 if [ "$confirm" != "y" ]; then
-  echo "Ok bye."
+  echo "Ok bye. 💁"
```

Comment:
😂

Reply:
I was going to put "Bye, Felicia" but ultimately decided against it.